src/readii/feature_extraction.py

```diff
@@ -76,11 +77,12 @@ def cropImageAndMask(
 def singleRadiomicFeatureExtraction(
 	ctImage: sitk.Image,
 	roiImage: sitk.Image,
-	pyradiomicsParamFilePath: Optional[str] = "./src/readii/data/default_pyradiomics.yaml",
+	pyradiomicsParamFilePath: Optional[str | Path] = "./src/readii/data/default_pyradiomics.yaml",
```

Comment:
Don't need the default argument here since it was added in if statement below.
```suggestion
	pyradiomicsParamFilePath: Optional[str | Path] = None,
```